README.md

```diff
@@ -57,8 +62,8 @@ $ yarn -v
 $ java -version
 ```
 
-Your Node.js version should be `10.15.0` or greater, your `yarn` version should
-be `1.0.0` or greater, and your `java` version should be `1.8.0` or greater.
+Your `node` version should be between `10.15.0` and `16.6.0`, your `yarn` version should
```

Comment:
I personally prefer referring to it as `Node.js` in documentation as `node` is an overloaded term in programming and could cause confusion.

Reply:
See other resolved thread for more information.